src/typeGuards.ts

```diff
@@ -8,7 +8,7 @@ import {
 } from './types';
 
 export function isProduct(item: Item): item is Product {
-  return (item as Product).section === 'Products';
+  return item.section !== 'Search Suggestions' && (item as Product).data?.image_url !== undefined;
```

Comment:
What do you think about repurposing isSearchSuggestion utility here instead of `item.section !== 'Search Suggestions' ` to be `!isSearchSuggestion(item)`